src/htmlparser.js

```diff
@@ -52,7 +52,7 @@
   });
 
   // Empty Elements - HTML 4.01
```

Comment:
I was also tempted to remove the commented code with regards to block elements below...